app/jobs/unseen_comments_notification_job.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class UnseenCommentsNotificationJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    User.includes(:comments).where(comments: { seen: false }).distinct.each do |user|
+      comments = user.comments.where(seen: false).to_a
```

Comment:
user.comments.unseen

Reply:
why to_a ?

Reply:
because `serialize': Unsupported argument type: ActiveRecord::AssociationRelation (ActiveJob::SerializationError)

        raise SerializationError.new("Unsupported argument type: #{argument.class.name}") unless serializer